vite.config.ts

```diff
@@ -31,7 +33,7 @@ export default defineConfig({
     environment: 'node',
     setupFiles: './tests/setup.ts',
     coverage: {
-      provider: 'istanbul',
+      provider: 'v8',
```

Comment:
have you noticed any differences in test coverage/issues using v8?

Reply:
no but it's the recommended one